src/ui/screens/home/controller/ControllerView.qml

```diff
@@ -477,19 +477,15 @@ Item {
 
         onClicked: {
             if (!box.connectionInfoScreenVisible) {
-                Glean.interaction.speedTestTriggered.record({
+                Glean.interaction.startSpeedTestSelected.record({
                     screen: "main",
-                    action: "select",
-                    element_id: "speed_test"
                 });
             } else {
-                Glean.interaction.speedTestClosed.record({
-                    screen: connectionInfoScreen.state == "open-ready" ? "speed_test_result"
+                Glean.interaction.closeSelected.record({
+                    screen: connectionInfoScreen.state == "open-ready" ? "speed_test_completed"
                                 : connectionInfoScreen.state == "open-error" ? "speed_test_error"
                                     : connectionInfoScreen.state == "open-loading" ? "speed_test_loading"
                                         : "unexpected",
```

Comment:
Does "unexpected" follow the outcome's naming convention?

Reply:
It doesn't, but the naming convention is for screen names and this is not it. Not recording would be bad here, because we may want to check this in case we ever get issues with this data and want to investigate.

We should probably document in the standard that `"unexpected"` or some other value should be used when an error state is necessary 🤔 WDYT?

Reply:
Should it be something like "speed_test_unexpected" to follow the naming convention? A general "unexpected" seems OK too if that is acceptable to the Data team.

Reply:
I'd advocate for a general "unexpected" as an error state. The naming convention is for screens and the "unexpected" could be used for not-screen-related error states. I'll file a follow up for us to make an addendum to the telemetry standard for possible error states and how to deal with them, this wy we can bring Yeonjoo and Santiago more easily into the conversation. Let me know if you are ok with that approach and I'll go ahead and merge this.

Reply:
Done! See: VPN-5775.